nimbleclient/v1/restclient.py

```diff
@@ -77,14 +77,16 @@ class NimOSAPIClient:
         'witnesses': 'v1/witnesses',
     }
 
-    def __init__(self, hostname, username, password, app_name=None, job_timeout=60, port=5392):
+    def __init__(self, hostname, username, password, app_name=None, job_timeout=60, port=5392, tenant_aware=False):
         """Initialize a session to the NimOS REST API."""
-
+        if tenant_aware:
+            port = 443
```

Comment:
This should not be hardcoded, the user may still need to present the API through a reverse proxy or firewall forwarder on another port than 443. Only assume default port (443) if initialized with default port (5392).

```
if tenant_aware and port == 5392:
    port = 443
```